lib/linguist/languages.yml

```
@@ -1016,6 +1016,16 @@ ChucK:
   codemirror_mode: clike
   codemirror_mime_type: text/x-java
   language_id: 57
+Circom:
+  type: programming
+  ace_mode: java
+  codemirror_mode: clike
+  codemirror_mime_type: text/x-java
+  extensions:
+  - ".circom"
+  color: "#707575"
+  tm_scope: none
```

This needs to be set to the scope in the grammar:

```suggestion
  tm_scope: source.circom
```